custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -96,8 +96,10 @@ def _state_aggregation_query(self, state_id):
             start_month_string=start_month_string)
         not_registered = "(agg_availing.is_registered = 0 AND agg_availing.registration_date::date < {start_month_string})::boolean".format(
```

Comment:
this should not need to be coerced to a boolean. those expressions already return a boolean